server/routes.js

```diff
@@ -1,10 +1,38 @@
-const express = require('express');
+const router = require('express').Router();
+// const log = require('loglevel');
+// const validateRequest = (req, res, next) => {
+//   next();
+// };
+const {
+  stakeholderGet,
+  stakeholderGetAll,
+  stakeholderGetUnlinked,
+  stakeholderUpdateLink,
+  stakeholderPatch,
+  stakeholderPost,
+} = require('./handlers/stakeholderHandler');
+const { handlerWrapper } = require('./utils/utils');
 
-const router = express.Router();
-const { stakeholderGet } = require('./handlers/stakeholderHandler');
+router
+  .route('/links/:stakeholder_id/:acctStakeholder_id')
+  .get(handlerWrapper(stakeholderGetUnlinked))
+  .patch(handlerWrapper(stakeholderUpdateLink));
 
-const { handlerWrapper } = require('./utils/utils');
+router
+  .route('/links/:stakeholder_id')
+  .get(handlerWrapper(stakeholderGetUnlinked))
```

Comment:
The naming of this route seem counterintuitive to me, since we are not returning 'link' objects but rather stakeholders that are not linked to the current stakeholder.  

I think a route such as /stakeholders?no_relationship=:stakeholder_id is closer to correct here.  We are querying for stakeholder objects and filtering out those that do not have a link to the current object but are accessible to the admin panel user.